<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QPushButton" name="btn5">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn4">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btn9">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="btnMenos">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btn6">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="btn1">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="btnIgual">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnAC">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 141, 42);
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="btnMas">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="btn3">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="etiqueta">
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>48</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 5px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(139, 66, 226, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(74, 74, 74);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="btnPunto">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btnDivi">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn7">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn8">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btnPor">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btn2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="btn0">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 2px solid qlineargradient(spread:pad, x1:0.165, y1:0.279, x2:1, y2:1, stop:0.153409 rgba(78, 82, 217, 255), stop:0.926136 rgba(255, 184, 255, 255));
font: 290 16pt &quot;Microsoft YaHei Light&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
